client/components/pulsing-dot/index.jsx

```diff
@@ -5,9 +5,53 @@
  */
 
 import React from 'react';
+import { number } from 'prop-types';
 import classnames from 'classnames';
 
-export default function PulsingDot( { active } ) {
-	const className = classnames( 'pulsing-dot', { 'is-active': active } );
-	return <div className={ className } />;
+class PulsingDot extends React.Component {
+	timeout = null;
+
+	state = {
+		show: false,
+	};
+
+	componentDidMount() {
+		const { delay } = this.props;
+
+		if ( ! delay ) {
+			return;
+		}
```

Comment:
If there's no delay, don't you want to set `state.show` to `true` immediately?

Reply:
You're right!

Reply:
@flootr But you didn't fix it before merging did you? :wink:

Reply:
No 😕 I didn't see your comment. It's fixed in 27588